external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaRelationSuite.scala

```diff
@@ -268,6 +268,24 @@ abstract class KafkaRelationSuiteBase extends QueryTest with SharedSparkSession
     }, topic, 0 to 19)
   }
 
+  test("global timestamp provided for starting and ending") {
```

Comment:
Same test as `timestamp provided for ending, offset provided for starting` but using global timestamp instead.